tests/virt/thinp_monitor_test.py

```diff
@@ -214,23 +218,18 @@ def test_extend(tmp_config):
     vm.volume_monitor.on_block_threshold(
         'vdb', '/virtio/1', alloc, 1 * MiB)
     assert drv.threshold_state == BLOCK_THRESHOLD.SET
+    assert len(vm.cif.irs.extensions) == 0
```

Comment:
Empty lists evaluate as false, so this would also work:
```python
     assert not vm.cif.irs.extensions
```
Both are constant time and quite similar, but to me using the truth value of the list is more 'pythonic' I'd say. You can leave it as is anyway (so that it is also consistent with other similar assertions in this file). Changing it is not in the scope of this PR.

Reply:
This works, but in these tests we also do:

    assert len(vm.cif.irs.extensions) == 1

Checking for non-empty list is not the same, we want to verify that
one extension request was sent.

So keeping the length check is more consistent and clear.